src/store/configureStore.ts

```diff
@@ -33,11 +46,18 @@ const onSagaError = (error, errorInfo) => {
   } catch {
     // no-op
   }
+
+  // Try recording to amplitude
+  try {
+    amplitudeTrack(ERROR_PAGE, errorInfo)
+  } catch {
+    // no-op
+  }
 }
 
 // Can't send up the entire Redux state b/c it's too fat
 // for Sentry to handle, and there is sensitive data
-const statePruner = state => {
+const statePruner = (state: AppState) => {
```

Comment:
this should be RootState i think

Reply:
this doesn't work b/c it introduces a circular dep, and redux says best practices is to use typeOf in declaring root state:
https://redux.js.org/usage/usage-with-typescript#define-root-state-and-dispatch-types